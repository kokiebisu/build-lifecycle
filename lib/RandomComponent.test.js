import React from 'react';
import renderer from 'react-test-renderer';
import { RandomComponent } from '.';

it('renders correctly', () => {
  const tree = renderer.create(<RandomComponent />).toJSON();
  expect(tree).toMatchSnapshot();
});
