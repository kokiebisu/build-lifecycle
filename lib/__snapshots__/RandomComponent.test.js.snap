// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <style>
    [object Object]
  </style>,
  <div
    className="bg"
  >
    <h2
      className="text"
    >
      Hi I am a random component
    </h2>
  </div>,
]
`;
